elastic_service/store_information.go

```diff
@@ -0,0 +1,7 @@
+package elastic_service
+
+import "github.com/daniyakubov/book_service_n/models"
+
+type StoreInformation interface {
+	Info(key string) ([]models.Action, error)
+}
```

Comment:
Why StoreInformation returns ([]models.Action, error)?
IMO this interface is redunsant